### ISSUE TITLE
Resume should not be downloaded withoud data

### ISSUE THREAD

**Issue by Rahilsiddique** [Rahilsiddique/resume-builder#9]
**Is your feature request related to a problem? Please describe.**
Currently, on the home page user can download an empty / incomeplete resume which should not be happening.

**Describe the solution you'd like**
There are two ways one can solve the issue:

Disable the download button until the data has been filled by user.
Hide the download button until the data has been filled by user .

**Comment by rtewari056** [comment]
@Rahilsiddique Can I work on this issue?

**Comment by Rahilsiddique** [comment]
yes @rtewari056 sure go ahead, pls let me know if you need any kind of help 

also, we are running an event along with Hacktoberest called ["Hacking The Hacktoberest"](https://community-website-aek7b80zf-cerrituscoders.vercel.app/events/hacking-the-hacktoberfest). You can register for it from [here](https://bit.ly/hacking-the-hacktoberfest) to earn some extra swags. 
Join our community: https://bit.ly/cerritus-coders
